.gitignore

```diff
@@ -20,3 +20,4 @@ php-extensions-*.tar.bz2
 monicadump.sql
 .scannerwork/
 .sentry-release
+.idea/
```

Comment:
I don't know this directory, but if it's platform specific, you may want to add it to your personal global gitignore.
See https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
When you use the PhpStorm IDE, all settings for project are stored under the `.idea` folder